service/mysql.go

```diff
@@ -0,0 +1,27 @@
+package service
+
+import (
+	"strconv"
+
+	"github.com/18F/cf-db-connect/launcher"
+	"github.com/18F/cf-db-connect/models"
+)
+
+type mySQL struct{}
+
+func (p mySQL) Match(si models.ServiceInstance) bool {
+	return si.ContainsTerms("mysql")
+}
+
+func (p mySQL) Launch(localPort int, creds models.Credentials) error {
+	return launcher.StartShell("mysql", []string{
+		"-u", creds.GetUsername(),
+		"-h", "0",
+		"-p" + creds.GetPassword(),
+		"-D", creds.GetDBName(),
+		"-P", strconv.Itoa(localPort),
+	})
+}
+
+// singleton
+var MySQL = mySQL{}
```

Comment:
Is there any reason this might be a bad idea? Given that the service type structs aren't mutable, I don't _think_ it should be a problem, but would like a second opinion.